qa/L0_perf_nomodel/run_test.sh

```diff
@@ -64,6 +65,7 @@ PERF_CLIENT_PROTOCOL_ARGS="-i grpc -u localhost:8001" &&
 PERF_CLIENT_PERCENTILE_ARGS="" &&
     (( ${PERF_CLIENT_PERCENTILE} != 0 )) &&
     PERF_CLIENT_PERCENTILE_ARGS="--percentile=${PERF_CLIENT_PERCENTILE}"
+PERF_CLIENT_PERCENTILE_ARGS="$PERF_CLIENT_PERCENTILE_ARGS --shared-memory \"${SHARED_MEMORY}\""
```

Comment:
If you want to do it this way you need to generalize  the variable name, something like PERF_CLIENT_EXTRA_ARGS.